### ISSUE TITLE
Support for WCAG Conformance

### ISSUE THREAD

**Issue by fongsean** [aehrc/smart-forms#1006]
Ideally the components in this renderer should be conformant to [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG21/). 

At a bare minimum, all items in the MBS 715 form should be.

**Comment by brendan-cheong** [comment]
Hi @fongsean 

We're really looking forward to this as (our app needs to comply with WCAG 2.1 AA)[https://www.digitalhealth.gov.au/about-us/accessibility]. 'Happy to help identify the issues if it helps. Here're some items that we've noticed at a glance:

1. [Heading ranks](https://www.w3.org/WAI/tutorials/page-structure/headings/#heading-ranks)
2. [Control labelling](https://www.w3.org/WAI/tutorials/forms/labels)
3. [Focus after button activation](https://www.w3.org/WAI/ARIA/apg/patterns/button/#keyboardinteraction)

**Comment by brendan-cheong** [comment]
Hey @fongsean , we've got a few read-only items in our questionnaire (and may start using the [setFormAsReadOnly](https://smartforms.csiro.au/docs/api/smart-forms-renderer/interfaces/QuestionnaireStoreType#setformasreadonly) feature in future), and noticed that the disabled attribute rather than readonly attribute is set on text controls. We believe that they should be set to readonly as [the controls are significant enough to receive focus and the values are submitted with the form](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly#attribute_interactions). (N.B. A solution may not be straightforward for non-text controls as they do not have a readonly attribute.)
